website_sale_stock_control/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+==========================
+Website Sale Stock Control
+==========================
+
+This module extends standard e-commerce allowing block sale products and
+their variants without stock.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to products and create one of type "Stockable".
+#. Select an option in stock control field.
+#. Go to shop and if product do not have stock yo will see an icon.
```

Comment:
`Go to the shop and if the product doesn't have stock, you will see an icon indicating that.`